src/BTVNanoCommissioning/workflows/ctag_valid_jec.py

```diff
@@ -297,12 +302,12 @@ def process(self, events):
 
         if not isRealData:
             corrected_jets = self._jet_factory["mc"].build(
-                add_jec_variables(events.Jet, events.fixedGridRhoFastjetAll),
+                add_jec_variables(events.Jet, rho),
                 lazy_cache=events.caches[0],
             )
         else:
             corrected_jets = self._jet_factory["data"].build(
-                add_jec_variables(events.Jet, events.fixedGridRhoFastjetAll),
+                add_jec_variables(events.Jet, rho),
```

Comment:
Would we need to distinguish different data runs as you did in the previous workflows? 
A more general question: weren't the `_jec` workflows outdated by the `_sf` ones, as the latter now have a `--isJERC` flag? Correct me if I am wrong

Reply:
these `_jec` workflows are outdated, remove in the next commit.